boefjes/boefjes/plugins/kat_cve_finding_types/normalize.py

```diff
@@ -54,7 +55,9 @@ def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterable[OOI
 
     yield CVEFindingType(
         id=cve_finding_type_id,
-        description=english_description["value"],
+        description=unicodedata.normalize("NFKC", english_description["value"]).translate(
+            str.maketrans("‘’“”", "''\"\"")
```

Comment:
Why is this necessary? We should be able to save the description if it has unicode characters in it?

Reply:
Dont exactly know why that doesnt work, but this seems like a harmless fix. Then I can move on with more urgent things.

Reply:
As discussed today, I'm going to look into the root cause of this problem. I've assigned the PR to me.